doc/developers/advanced_installation.rst

```diff
@@ -0,0 +1,429 @@
+
+.. _advanced-installation:
+
+===================================
+Advanced installation instructions
+===================================
+
+There are different ways to get scikit-learn installed:
+
+  * Install the version of scikit-learn provided by your
+    :ref:`operating system or Python distribution <install_by_distribution>`.
+    This is the quickest option for those who have operating systems that
+    distribute scikit-learn.
+
+  * :ref:`Install an official release <install_official_release>`. This
+    is the best approach for users who want a stable version number
+    and aren't concerned about running a slightly older version of
+    scikit-learn.
+
+  * :ref:`Install the latest development version
+    <install_bleeding_edge>`. This is best for users who want the
+    latest-and-greatest features and aren't afraid of running
+    brand-new code.
+
+.. note::
+
+    If you wish to contribute to the project, you need to
+    :ref:`install the latest development version<install_bleeding_edge>`.
+
+
+.. _install_official_release:
+
+Installing an official release
+==============================
+
+Scikit-learn requires:
+
+- Python (>= 2.6 or >= 3.3),
+- NumPy (>= 1.6.1),
+- SciPy (>= 0.9).
+
+
+Mac OSX
+-------
+
+Scikit-learn and its dependencies are all available as wheel packages for OSX::
+
+    pip install -U numpy scipy scikit-learn
+
+
+Linux
+-----
+
+At this time scikit-learn does not provide official binary packages for Linux
+so you have to build from source.
```

Comment:
You should mention here that various distributions have scikit-learn packages, although they might not be up to date with the latest version of scikit-learn.